core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/Group.java

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+package org.wso2.carbon.user.core.common;
+
+import java.util.Map;
+
+/**
+ * Represents a Group.
+ */
+public class Group extends AbstractEntity {
```

Comment:
The permission is also an attribute of the Group

Reply:
Added